modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -223,8 +223,8 @@ public String getContainerId() {
 		DDMFormRenderingContext ddmFormRenderingContext =
 			createDDMFormRenderingContext(ddmForm);
 
-		ddmFormRenderingContext.addProperty("formInstanceId",
-			ddmFormInstance.getFormInstanceId());
+		ddmFormRenderingContext.addProperty(
+			"formInstanceId", ddmFormInstance.getFormInstanceId());
```

Comment:
@linolaoj, shouldn't you use the method `setDDMFormInstanceId` that you created on this commit?